tests/test_distribution_validation.py

```
@@ -89,14 +89,17 @@ class TestAliasVsReferenceDistribution:
     the rank-indexed empirical CDFs should fail to reject."""
 
     def test_first_names_match_reference_numpy(self, tmp_path: Path):
+        """
+        Verifies that sampling from the AliasTable reproduces NumPy's categorical sampling for first names in period 'ate2000'.
+        
+        Draws 20,000 samples from the partitioned alias table and from numpy.random.choice using the same RNG seed, maps alias draws to integer positions, and compares the two empirical distributions with a two-sample Kolmogorov–Smirnov test; the test fails if the KS p-value is less than or equal to 0.01.
```

Ruff (0.15.11)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)